### PR TITLE
[GEOS-7544] Use File for Importer (not Resource)

### PR DESCRIPTION
**Pull request by tbarsballe**
For https://osgeo-org.atlassian.net/browse/GEOS-7544

Could I get a review @jodygarnett or @NielsCharlier ?

### CODE REVIEW

File: src/extension/importer/bdb/src/main/java/org/geoserver/importer/bdb/BDBImportStore.java [hunk 1]
```diff
@@ -21,7 +21,6 @@
 import org.geoserver.importer.ImportStore;
 import org.geoserver.importer.ImportTask;
 import org.geoserver.importer.Importer;
-import org.geoserver.platform.resource.Resource;
```
Comment (jodygarnett) [inline src/extension/importer/bdb/src/main/java/org/geoserver/importer/bdb/BDBImportStore.java]: You will need to update the header on this file.

File: src/extension/importer/core/src/main/java/org/geoserver/importer/DataStoreFormat.java [hunk 2]
```diff
@@ -224,7 +223,7 @@ public DataStore createDataStore(ImportData data) throws IOException {
 
             if (f != null) {
                 Map<String,Serializable> map = new HashMap<String, Serializable>();
-                map.put("url", relativeDataFileURL(Resources.find(f).toURI().toURL().toString(), catalog));
+                map.put("url", relativeDataFileURL(f.toURI().toURL().toString(), catalog));
```
Comment (jodygarnett) [inline src/extension/importer/core/src/main/java/org/geoserver/importer/DataStoreFormat.java]: The DataUtilities method is slightly more stable then toURI().toURL() on platforms like windows.

**Comment by tbarsballe** [comment]
Updated to fix compile failures caused by revert, added headers to BDBImportStore

**Comment by jodygarnett** [comment]
Thanks @tbarsballe - running checks now and will merge.

**Pull request closed without merging**